### DISCUSSION TITLE
Tempo de Verificação de lotes

### DISCUSSION THREAD

**Discussion by ABSPereira** [Chia-Network/chia-blockchain#6187] [Support]
Estou com uma dúvida nova aqui:
O FAQ Chia. Diz que as tentativas de verificação devem ter menos de 5 segundos entre elas. Melhor se for menos de 2 segundos . 
Mas meu programa está fazendo em média 10 segundos entre elas . 
o que eu devo fazer para diminuir esse tempo? 
![chioa](https://user-images.githubusercontent.com/84117989/120037301-cc379280-bfd7-11eb-9cf3-8e60de0f32ed.PNG)

**Comment by BrandtH22** [comment]
@ABSPereira , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it is expected that challenges/signage points are received at slightly different intervals due to network latency and peer connection quality. This is only a concern if your farm is unable to remain in sync or if there are errors in the logs.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
